Protocol/Tests/Protocol/QActions/QAction/CSharpCoreInterAppBrokerSupport.cs

```diff
@@ -130,21 +111,22 @@ private void CheckForSendMessage(CallingMethodClass callingMethod)
             if (expressionOfArgument is IdentifierNameSyntax identifierName)
             {
                 // Someone passing along a different variable possibly defined earlier as the AgentId.
-                var symbolInfo = semanticModel.GetSymbolInfo(identifierName).Symbol;
+                var symbolInfo = RoslynHelper.GetSymbol(identifierName, semanticModel);
 
                 if (IsArgumentPropertyFromReturnAddress(symbolInfo))
                 {
-                    results.Add(Error.InvalidInterAppReplyLogic(test, qAction, qAction, "Message", "Send(ReturnAddress", qAction.Id.RawValue));
+                    results.Add(Error.InvalidInterAppReplyLogic(test, qAction, qAction, "Message", "Send(ReturnAddress", qAction.Id.RawValue)
+                                     .WithCSharp(callingMethod));
```

Comment:
.WithCSharp  did not work when the compilation did not work.

Reply:
WithCSharp is purely for the navigation in DIS. Not sure what you mean with 'did not work'